test/run_tests.sh

```diff
@@ -5,7 +5,8 @@
 
 SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
 TESTS_SMT_COMP_DIR="${SCRIPT_DIR}/smt-comp"
-EXPECT_SMT_COMP_DIR=${TESTS_SMT_COMP_DIR}/expect
+TESTS_NON_SMT_COMP_DIR="${SCRIPT_DIR}/extensions/non-smtcomp"
+TESTS_Z3_DIR="${SCRIPT_DIR}/extensions/z3"
```

Comment:
You will now have to add checks if those two directories (and the corresponding `expect` directories) exist, similar to how it is done for the smt-comp directory on master.

Reply:
Sorry, missed that. Added now.